### ISSUE TITLE
setSpeakMiddleware not creating base element correctly

### ISSUE THREAD

**Issue by alopix** [microsoft/botframework-solutions#2704]
#### What project is affected?
`botbuilder-solutions`

#### What language is this in?
`typescript`

#### What happens?
`createBaseElement` overwrites the `speak` element with the `text` element.
```
result.elements = [
    {
        type: 'text',
        text: value
    }
];
```

should be 

```
result.elements[0].elements = [
    {
        type: 'text',
        text: value
    }
];
```

#### What are the steps to reproduce this issue?
-

#### What were you expecting to happen?
-

#### Can you share any logs, error output, etc.?
Otherwise `addVoiceElementIfMissing` will always throw `rootElement undefined`.

#### Any screenshots or additional context?
-

**Comment by Batta32** [comment]
Hi @alopix, thanks for reporting the issue!

We managed to validate the scenario in which the `createBaseElement` is overwriting the `speak` element with the `text` element.

We will take into account your suggestion and get back to you later.

**Comment by Batta32** [comment]
@alopix, we pushed our changes in the following brach: [feature/southworks/setspeakmiddleware-issue](https://github.com/microsoft/botframework-solutions/tree/feature/southworks/setspeakmiddleware-issue). Feel free to test the solution and share the repro steps that you follow 😊.

![image](https://user-images.githubusercontent.com/11904023/69273784-cb8a9880-0bb7-11ea-8e50-e71bdd4b1ac0.png)

Also, we tried your suggestion but we noticed that we should add a validation due to `elements` property could be `Array<Element> | undefined`, so we decided to unify all the initialization in one step.

![image](https://user-images.githubusercontent.com/11904023/69273796-d3e2d380-0bb7-11ea-9883-d3a79534065d.png)

We will be attentive to your answer 😊.

**Comment by alopix** [comment]
Yes, thanks, this seems to work, was basically how I copy-paste-fixed it locally as well 👍🏼

**Comment by Batta32** [comment]
Awesome @alopix! This issue will be closed with the PR #2751
